AEPServices/Sources/log/LoggingService.swift

```diff
@@ -50,6 +50,6 @@ class LoggingService: Logging {
     // MARK: Logging
 
     func log(level: LogLevel, label: String, message: String) {
-        os_log("%@", log: osLog(label), type: osLogType(level), message)
+        os_log("%@", log: osLog("AEP SDK : \(label)"), type: osLogType(level), message)
```

Comment:
i think let's change this to:
```
AEP SDK \(level.uppercased()) - <\(label)>
```

Reply:
@sbenedicadb  Done, the following log is in the new format: 

`2020-09-25 15:28:55.231320-0500 E2ETesting[57955:1712641] [AEP SDK TRACE - <SQLiteDataQueue>] Database record did not have valid data: SELECT min(id),uniqueIdentifier,timestamp,data FROM TB_AEP_DATA_ENTITY;.`